lib/rpc/modules/eth.js

```diff
@@ -0,0 +1,15 @@
+class Eth {
+  constructor (chain) {
+    this._chain = chain
+  }
+
+  getBlockByNumber (params, cb) {
+    let [blockNumber] = params
+    blockNumber = Number.parseInt(blockNumber, 16)
+    this._chain.getBlock(blockNumber, (err, block) => {
+      cb(err, block.toJSON(true))
+    })
+  }
+}
```

Comment:
Which part do you want to replicate from `ganache-core`? 

took a look I feel we should expand argument like `function(block_number, include_full_transactions, callback)` instead of `(params, cb)` to make it more clear. 

I'm not familiar with `ganache-core` codebase, so I don't have any opinion for that yet, but I agree we should analyze `ganache-core` deeper to get a better idea to implement rpc.